project/Dependencies.scala

```diff
@@ -13,37 +13,45 @@
 import sbt._
 
 object Dependencies {
+
+  val ResolutionRepos = Seq(
+    // Speed-up build
+    "snowplow" at "https://snowplow.bintray.com/snowplow-maven"
+  )
+
   object V {
     // Scala Snowplow
-    val analyticsSdk       = "0.3.2"
-    val igluClient         = "0.5.0"
-    val igluCore           = "0.4.0"
+    val analyticsSdk       = "0.4.2-M3"
+    val igluClient         = "0.6.0-M7"
+    val igluCore           = "0.5.0"
     val processingManifest = "0.1.0"
```

Comment:
Can we remove this?

Reply:
What should exactly remove ?

Reply:
Whole processing manifest dependency. It seems we're not using it anymore as 0.1.0 is binary and source incompatible with most of our artifacts and it would have failed.

Reply:
Okay then, will remove it from respective commit